crypto/crypto.go

```diff
@@ -90,28 +98,38 @@ func HashData(kh KeccakState, data []byte) (h common.Hash) {
 
 // Keccak256 calculates and returns the Keccak256 hash of the input data.
 func Keccak256(data ...[]byte) []byte {
-	b := make([]byte, 32)
-	d := hasherPool.Get().(KeccakState)
+	b := Keccak256Hash(data...)
+	return b[:]
+}
+
+// Keccak256RLP calculates and returns the Keccak256 hash of the input interface.
+func Keccak256RLP(x interface{}) []byte {
```

Comment:
I don't think we need both RLP and RLPHash tbh

Reply:
I Agree, it's similar to `Keccak256`  and `Keccak256Hash`. How about just keep `XXHash` methods?